src/lib/networksRegistry.js

```diff
@@ -14,24 +14,8 @@ const testNetwork = {
   compilerUrl: 'https://compiler.aepps.com',
 };
 
-let networks = process.env.NODE_ENV === 'production' ? [
-  mainNetwork,
-  testNetwork,
-] : [
-  testNetwork,
-  mainNetwork,
-];
-
-export const defaultNetwork = process.env.VUE_APP_NETWORK_NAME ? {
-  name: process.env.VUE_APP_NETWORK_NAME,
-  url: process.env.VUE_APP_NODE_URL,
-  middlewareUrl: process.env.VUE_APP_MDW_URL,
-  explorerUrl: process.env.VUE_APP_EXPLORER_URL,
-  compilerUrl: process.env.VUE_APP_COMPILER_URL,
-} : mainNetwork;
-
-if (process.env.VUE_APP_NETWORK_NAME) {
-  networks = [defaultNetwork];
-}
+const networks = (window.overrideNetwork && [window.overrideNetwork])
```

Comment:
Have written Perl? You really love nested ternary conditions :)

Reply:
This is also because I prefer `const` to `let`. I did a webshop in Perl at university, but I didn't remember it, maybe it is unconscious 😃